### DISCUSSION TITLE
Running Talos in lxc containers ( on Proxmox )

### DISCUSSION THREAD

**Discussion by andrnils** [siderolabs/talos#8417] [Q&A]
Hello!
I'm considering setting up a Talos Linux cluster at work. We run our infrastructure on Proxmox virtualisation, primarily lxc-containers.
Can Talos be installed in an LXC container? Or does it strictly depend on its own kernel?
 How would storage be handled in lxc?

**Comment by smira** [comment]
Talos can run in a container, which is not a full Talos experience (using host Linux kernel): https://www.talos.dev/v1.6/talos-guides/install/local-platforms/docker/#running-talos-in-docker-manually

Storage can be only mounted as volumes from the host, and later re-used in Kubernetes, e.g. via https://www.talos.dev/v1.6/kubernetes-guides/configuration/local-storage/
